dashbuilder/image.yaml

```diff
@@ -3,7 +3,7 @@ schema_version: 1
 name: "rhpam-7/rhpam-dashbuilder-rhel8"
 description: "Red Hat Process Automation Manager Standalone Dashbuilder 7.10 container image"
 version: "7.10.0"
-from: "ubi8-minimal:8-released"
+from: "registry.redhat.io/ubi8/ubi-minimal:latest"
```

Comment:
Why do we have two different images? (`registry.redhat.io/ubi8/ubi-minimal:latest` and `registry.redhat.io/ubi8/ubi:latest`). Is it possible to keep all same, possibly the minimal one if works?

Reply:
This image is new and we start with minimal, the existing image was kept the ub8 only, but it is on my queue to migrate it to ubi8-minimal.
I'll try to do migrate the images before 7.10. Let's see :)

Reply:
but for now, we'll not change the other images to the ubi8-minimal.

Reply:
Ah okay 🙂